docs/containers/app-service.md

```diff
@@ -1,14 +1,20 @@
 ---
 Order: 8
 Area: containers
-TOCTitle: Deploy to Azure
+TOCTitle: Deploy Containers
 ContentId: 044913F5-F99D-4228-A916-0443260AB7FB
 PageTitle: Deploy a containerized app to Azure App Service
 DateApproved: 10/28/2020
 MetaDescription: Using Visual Studio Code, build a container image for your application, push the image to a container registry, and deploy to Azure App Service.
 ---
 # Deploy to Azure App Service
```

Comment:
I think you are asking if the title on line 10 should be changed to match the TOC Title. If that's the case, the reason I made them different is so that the title before the tutorial would be more specific. It's kind of the same idea I went with for Java and Azure. The TOC titles are "Deploy X apps" but the article titles are "Deploy X Web Apps". But if you want the TOC and article titles to match, then I'm happy to make that change